### PR TITLE
Rep Console Update

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Updates the Rep Console to have some basic sales stuff, and the sales works as intended

## Why It's Good For The Game
More........ MORE!

## Changelog
:cl:
add: Added stuff for reps to sell
/:cl:

### CODE REVIEW

File: ModularTegustation/representative_console.dm [hunk 1]
```diff
@@ -312,44 +328,120 @@
...
-/datum/data/lc13research/syringe/ResearchEffect(obj/structure/representative_console/caller)
+/datum/data/lc13research/injector/ResearchEffect(obj/structure/representative_console/caller)
 	new /obj/item/reagent_containers/syringe/epinephrine(get_turf(caller))
 	ItemUnlock(caller.order_list, "K Corp Implanter ",	/obj/item/implanter, 50)
 	ItemUnlock(caller.order_list, "K Corp Chem Implant ",	/obj/item/implantcase/chem, 300)
 	..()
 
+/datum/data/lc13research/krevival
+	research_name = "K Corp Experimental Ampule "
+	research_desc = "Hey, listen buddy, it's Joe from research. <br>The supply team went home for the night, and I'm tired, but I need this tested. <br>It'll revive one guy, once. It's a major breakthrough, give it a shot."
```
Comment (LanceSmites328) [inline ModularTegustation/representative_console.dm:358]: Joe K-Corp

File: ModularTegustation/representative_console.dm [hunk 2]
```diff
@@ -312,44 +328,120 @@
...
+
 //-----L_CORP-----
+//L Corp generally makes your life a lot easier with L-Corp related things.
 
-/datum/data/lc13research/detector
-	research_name = "Affordable Detector Research"
-	research_desc = "Contributed energy will be sent to welfare in order to develop <br>abnormality detectors made from more affordable material."
+/datum/data/lc13research/reroll
+	research_name = "Reroll Abno Choices "
+	research_desc = "You can PROBABLY convince HQ Extraction to give you another set of 3 abnos, as you're pretty tight with Big B. "
```
Comment (LanceSmites328) [inline ModularTegustation/representative_console.dm:371]: 128ba089da7f53c7fa19add35be1e012e3fde76007c0d6b904707e51a91010d6_1

File: ModularTegustation/tegu_items/representative/research.dm
Comment (LanceSmites328) [inline ModularTegustation/tegu_items/representative/research.dm:1]: Holds duplicate research to that in representative_console.dm

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: ModularTegustation/tegu_items/representative/research.dm
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply ModularTegustation/tegu_items/representative/research.dm:1]: Ah, will be removed in a bit

**Review by LanceSmites328 — COMMENTED** [review]
Let me know when merge conflicts are fixed

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**